packages/manager-components/src/components/tile/tile.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { ScTile, ScTileProps } from './tile';
```

Comment:
**question:** What does `Sc` indicate? Is it a super-component?

Reply:
Did we not agree that we will not have Prefix to the components name?

Reply:
And even if we do so, `Sc` doesn't make sense to me. It can be `Mc` to indicate "Manager Components"

Reply:
Fixed